waymo_raw_data_to_argoverse.py

```diff
@@ -84,17 +101,28 @@ def save_json_dict(json_fpath: Union[str, "os.PathLike[str]"], dictionary: Dict[
 	with open(json_fpath, "w") as f:
 		json.dump(dictionary, f)
 
+def get_log_id_from_files(files, record_dir):
```

Comment:
type hints would be nice here. Do you mind also adding a 1-line docstring explaining the method's purpose, and then a bit of a longer docstring explaining the input string and output string format

let's use Google-style docstrings. Do you mind reading through this section?
https://google.github.io/styleguide/pyguide.html#381-docstrings

Reply:
Addressed in [this commit](https://github.com/johnwlambert/waymo_to_argoverse/pull/2/commits/8931614c713c462a1efbbc0f06a366d44a28df25)